services/nginx/.gitignore

```diff
@@ -0,0 +1,2 @@
+default.conf
```

Comment:
I think we should add a `.gitignore` file in the generator. Added an issue for that, #40 .